src/QueryPressure.MySql.App/MySqlProviderInfo.cs

```diff
@@ -0,0 +1,19 @@
+using QueryPressure.Core;
+
+namespace QueryPressure.MySql.App;
+
+public record MySqlProviderInfo()
+  : ProviderInfo("MySql", new[] { "mysql", "sql" }, Script)
+{
+  // source: https://github.com/microsoft/monaco-editor/blob/main/website/src/website/data/home-samples/sample.mysql.txt
+  private const string Script = @"-- Sample
```

Comment:
Raw String Literals are cool for SQL readability

Reply:
done